tests/test_bvmf_calendar.py

```diff
@@ -0,0 +1,72 @@
+from unittest import TestCase
+import pandas as pd
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.exchange_calendar_bvmf import BVMFExchangeCalendar
+
+
+class BVMFCalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
```

Comment:
Should we add a test case for the special open day (`QuartaCinzas`)?

Reply:
yes, will do. I didn't see that special open day.